Mathlib/MeasureTheory/Measure/HausdorffProjectionLe.lean

```diff
@@ -0,0 +1,97 @@
+/-
+Copyright (c) 2025 Francesco Nishanil Chotuck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Francesco Nishanil Chotuck
+-/
+
+import Mathlib.MeasureTheory.Measure.Haar.Disintegration
+import Mathlib.MeasureTheory.Measure.Hausdorff
+
+/-!
+# Orthogonal Projections and Hausdorff Measure
+
+This file proves that Hausdorff measure is non-increasing under orthogonal projection
+onto a finite-dimensional Euclidean subspace.
+
+## Main results
+
+* `hausdorffMeasure_orthogonalProjection_le`: The `s`-dimensional Hausdorff
+  measure of the projection of a set is at most the measure of the original set.
+
+## References
+
+This formalisation is based on Lemma 3.5 from:
+
+- Julian Fox, *Besicovitch Sets, Kakeya Sets, and Their Properties*,
+  University of Chicago REU, 2021.
+  https://math.uchicago.edu/~may/REU2021/REUPapers/Fox.pdf
+-/
+
+open ENNReal MeasureTheory Submodule
+
+/--
+In a finite-dimensional Euclidean space, the norm of the orthogonal projection
+of a vector `v` onto a subspace `V` is less than or equal to the norm of `v`.
+-/
+lemma norm_orthogonalProjection_le (n : ℕ)
+  (V : Submodule ℝ (EuclideanSpace ℝ (Fin n)))
+  (v : EuclideanSpace ℝ (Fin n)) :
+    ‖orthogonalProjection V v‖ ≤ ‖v‖ := by
+  -- Use the definition of operator norm to bound the projection of v
+  have h₁ : ‖orthogonalProjection V v‖ ≤ ‖orthogonalProjection V‖ * ‖v‖ := by
+    apply ContinuousLinearMap.le_opNorm
+  -- Use the known fact that the operator norm of an orthogonal projection is at most 1
+  have h₂ : ‖orthogonalProjection V‖ ≤ 1 := by
+    apply orthogonalProjection_norm_le
+  -- Combine the inequalities to obtain the final bound
+  have h₃ : ‖orthogonalProjection V‖ * ‖v‖ ≤ ‖v‖ := by
+    calc
+      ‖orthogonalProjection V‖ * ‖v‖ ≤ 1 * ‖v‖ := by
+        gcongr
+      _ = ‖v‖ := by
+        simp only [one_mul]
+  exact le_trans h₁ h₃
+
+/--
+The orthogonal projection onto a subspace `V` in a finite-dimensional Euclidean space
+is a `1`-Lipschitz function.
+-/
+lemma lipschitzWith_orthogonalProjection (n : ℕ)
```

Comment:
I think this should also be in a different file: the idea is that if someone is looking for the fact that projection is 1-lipschitz, it doesn't quite make sense to find it in a measure theory file!